packages/coordinator/tsconfig.json

```diff
@@ -5,7 +5,8 @@
     "allowSyntheticDefaultImports": true,
     "target": "es6",
     "lib": [
-      "es2019"
+      "es2019",
+      "DOM"
```

Comment:
DOM potentially add to resolve build issues- check if removing it makes a difference.

Reply:
DOM is required for some new libraries. Some classes such as Algorithm, BufferSoruce and CryptoKey are used by libraries and are missing without this DOM inclusion.